recipes/static/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "static" %}
+{% set version = "1.1.1" %}
+{% set md5 = "75c738ed25cc69a71299b93369ee4e40" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+    preserve_egg_dir: True
```

Comment:
Yeah, it is. Though I have seen a package or two do this when generating skeletons from PyPI. If I had to guess, [this]( https://github.com/lukearno/static/blob/master/setup.py#L31 ) might be the reason. In general, the [docs]( http://conda.pydata.org/docs/building/meta-yaml.html#preserve-python-egg-directory ) say some setuptools features cause this. Though I'm not sure what other ones would off the top of my head.